Wikipedia/Code/AppearanceSettingsViewController.swift

```diff
@@ -126,6 +124,7 @@ open class AppearanceSettingsViewController: UIViewController, UITableViewDataSo
             case Theme.dark:
                 cell.disclosureSwitch.isEnabled = true
                 cell.disclosureSwitch.addTarget(self, action: #selector(self.handleImageDimmingSwitchValueChange(_:)), for: .valueChanged)
+                userDidSelect(theme: currentAppTheme.withDimmingEnabled(cell.disclosureSwitch.isOn))
```

Comment:
without this, after changing the theme from Dark (image dimming on) to Sepia or Default and changing back to Dark, the example image loses the dim:

![img_0022](https://user-images.githubusercontent.com/9299317/28902306-19aa01c2-77cc-11e7-80de-2e815826ffd2.PNG)